workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginInstallContent.tsx

```diff
@@ -69,25 +148,179 @@ export const MarketplacePluginInstallContent = ({
     }
   }, [codeEditor, copyToClipboard]);
 
+  const navigate = useNavigate();
+  const theme = useTheme();
+  const borderColor = theme.palette.mode === 'dark' ? '#a3a3a3' : '#c7c7c7';
+  const installationInstructions = plugin.spec?.installation;
+  const examples = packages[0]?.spec?.appConfigExamples;
+  const showRightCard = installationInstructions || examples;
+  // const [yamlText, setYamlText] = useState('');
```

Comment:
YAML state isn't needed I spend the day yesterday really on having an uncontrolled (for performance) editor. :)